Bottom: 42b565d91d0c22bb50d687c9c488264c042e3eec
Top:    f52ddb37f0c20f25e84d5641bed2aae05fab6f6d
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2016-01-06 17:27:03 +0000

borast: Win32: Comment #define in borast-compiler-private.h


---

diff --git a/src/borast/borast-compiler-private.h b/src/borast/borast-compiler-private.h
index f886f40..165b42e 100644
--- a/src/borast/borast-compiler-private.h
+++ b/src/borast/borast-compiler-private.h
@@ -160,7 +160,7 @@
 #endif
 
 #if (defined(__WIN32__) && !defined(__WINE__)) || defined(_MSC_VER)
-#define snprintf _snprintf
+//#define snprintf _snprintf
 #define popen _popen
 #define pclose _pclose
 #define hypot _hypot
